docs/setup/aks/getting-started-aks.md

```diff
@@ -174,16 +178,22 @@ kubectl port-forward -n nuclio $(kubectl get pods -n nuclio -l nuclio.io/app=das
 kubectl port-forward -n kube-system $(kubectl get pod -n kube-system -l k8s-app=traefik-ingress-lb -o jsonpath='{.items[0].metadata.name}') 8080:80
 ```
 
+<a id="deploy-a-function-with-the-nuclio-dashboard"></a>
 ## Deploy a function with the Nuclio dashboard
 
-Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation). You should see the [Nuclio dashboard](/README.md#dashboard) UI. Choose one of the built-in examples and click **Deploy**. The first build will populate the local Docker cache with base images and other files, so it might take a while, depending on your network. When the function deployment is completed, you can click **Invoke** to invoke the function with a body.
+Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation) to see the [Nuclio dashboard](/README.md#dashboard).
+Select the "default" project and then select **New Function** from the action toolbar to display the **Create function** page (http://localhost:8070/projects/default/create-function).
+Choose one of the predefined template functions, and select **Deploy**. 
+The first build populates the local Docker cache with base images and other files, so it might take a while to complete, depending on your network.
+When the function deployment completes, you can select **Invoke** to invoke the function with a body.
```

Comment:
Invoke button is not there anymore, do you mean _Test_?

Reply:
>     2. this should be merged after #1872 due to changes made on minikube docs

I merged the PR #1872 to the current PR branch and made some manual edits & related similar rephrasing edits to the Nuclio CLI prerequisite bullet in other GS docs — see commit 712a239.

 >     1. ~invoke~ button does not exists, you mean _test_ button (e.g. select _test_ button)

As part of the PR #1727 merge, the reference to the **Invoke** button was replaced with **Test** (+ I replaced "click" with "select", as I had asked for in PR #1727; when doing the merge I found that we already use "select" in multiple locations in this paragraph, probably also because of my previous edits for the current PR).